v21.1/operate-cockroachdb-kubernetes.md

```diff
@@ -542,6 +543,59 @@ If you previously [authenticated with `cockroach cert`](#example-authenticating-
     cockroachdb-1                         1/1     Terminating   0          4h16m
     cockroachdb-2                         1/1     Running       0          43s
     ~~~
+
+## Secure the webhooks
+
+The Operator ships with both [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) and [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) webhooks. Communication between the Kubernetes API server and the webhook service must be secured with TLS.
+
+By default, the Operator searches for the TLS secret `cockroach-operator-webhook-ca`, which contains a CA certificate. If the secret is not found, the Operator auto-generates `cockroach-operator-webhook-ca` with a CA certificate for future runs.
+
+The Operator then generates a one-time server certificate for the webhook server that is signed with `cockroach-operator-webhook-ca`. Finally, the CA bundle for both mutating and validating webhook configurations is patched with the CA certificate.
+
+You can also use your own certificate authority rather than `cockroach-operator-webhook-ca`. Both the certificate and key files you generate must be PEM-encoded. See the following [example](#example-using-openssl-to-secure-the-webhooks).
+
+### Example: Using OpenSSL to secure the webhooks
+
+These steps demonstrate how to use the [`openssl genrsa`](https://www.openssl.org/docs/manmaster/man1/genrsa.html) and [`openssl req`](https://www.openssl.org/docs/manmaster/man1/req.html) subcommands to secure the webhooks on a running Kubernetes cluster:
```

Comment:
Should we add a note here about this needing to be done in the `cockroach-operator-system` namespace?

Reply:
I realized we should have this as a callout on all pages using `kubectl` steps with the Operator. I created an include that now appears at the top of each page. 

For the webhook steps specifically, one of the steps also reiterates that you need to check the Operator namespace. Let me know if this all looks ok.

Reply:
@pseudomuto Let me know if this solution looks ok to you!